libs/web/styles/src/lib/_general.scss

```diff
@@ -1,5 +1,12 @@
-html,
-body {
-  height: 100vh;
-  // background-color: green;
-}
\ No newline at end of file
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500&display=swap');
```

Comment:
I am not used to separating the declarations as so. What is the motivation behind this again?

Reply:
We can pause this task while we resolve [this discussion](https://github.com/Sage-Bionetworks/challenge-registry/issues/137#issuecomment-1101602799). 

This is the way theme is implemented by most of the apps I've seen that use Angular Material. See [Theming Angular Material](https://material.angular.io/guide/theming) for background.

> Angular Material's theming system lets you customize color and typography styles for components in your application. The theming system is based on Google's [Material Design](https://material.io/design/material-theming/overview.html) specification.

Reply:
I'll still look for information on the benefits of the theming solution that Google came up with. If this solution clashes with generated SCSS content, dropping the current theming solution is an option.

Reply:
Oh, I see!

Would it be too much to propose creating one general stylesheet that all the apps will use as a base? This stylesheet would declare everything, e.g. grids, sizes, typography, and color scheme. That way, for any apps that do not have time/want to create their own themes, their pages can still look polished?

Reply:
@vpchung Here is a quick intro to the "how" [Angular Material (AM) theming system works](https://material.angular.io/guide/theming-your-components).

Reply:
One motivation for using AM theming system that can already be mentioned is to support multiple themes, e.g. a light and dark theme. The solution enables the client to load only the theme they needs, which is defined by the palette colors and typography properties.

Reply:
Here is an example of third-party style and/or theme for Angular: https://www.telerik.com/kendo-angular-ui/components/styling/. This project may teach us about how to manage styles and/or themes.

Reply:
Ohh, I see now that the approach to styling has changed a lot in the recent years - makes sense though.  I will see if it will be easy to split what teleportHQ spits out.

Reply:
I am also looking at [this](https://www.smashingmagazine.com/2020/04/dark-mode-react-apps-styled-components/#comments-dark-mode-react-apps-styled-components), though this is for React. I imagine the general process to be the same though.

Reply:
> Would it be too much to propose creating one general stylesheet that all the apps will use as a base? This stylesheet would declare everything, e.g. grids, sizes, typography, and color scheme. That way, for any apps that do not have time/want to create their own themes, their pages can still look polished?

It does not seem like a good practice to add component-specific code in a global document. The current solution implemented is the one officially recommended. It also enables inheritance 1) for components in a given app and 2) for components in different apps. Could you evaluate the gap between the current solution and the code generated by Figma?

Reply:
^ yes, I proposed that approach before I learned about the shift to styled components.